static/app/components/onboarding/gettingStartedDoc/utils/feedbackOnboarding.tsx

```diff
@@ -82,3 +82,44 @@ export const getFeedbackConfigOptions = ({
   }
   return options.join('\n');
 };
+
+export const getCrashReportModalIntroduction = () =>
+  t(
+    'Collect feedback on your errors by installing our crash-report modal. This allows users to submit feedback after they experience an error via an automatic modal that pops up after an error occurs. The default modal will prompt the user for their name, email address, and description of what occurred.'
+  );
+
+export const getCrashReportModalInstallDescriptionJavaScript = () =>
+  tct(
+    'You can collect feedback right before the event is sent, using [code:beforeSend].',
```

Comment:
technically the event is already sent by the time the modal shows up. Should reword slightly?

```suggestion
    'You can collect feedback at the time the event is sent, using [code:beforeSend].',
```